src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -926,8 +928,10 @@ def toIODAobs(self, OutDir, TotalBias=False, clobber=True, platforms=None):
 
                 if v == 'bend':
                     # sort record_number
-                    record_number = self.var('record_number')[idx]
-                    id_recordnum_sort = sorted(range(len(record_number)), key=record_number.__getitem__)
+                    idx = range(self.nobs)
+                    ### record_number = self.var('record_number')[idx]
+                    ### record_number = self.var('sequenceNumber')[idx]
```

Comment:
this `sequenceNumber` line I added as well as thought this may work in replacement for `record_number` but did not work